From de060704eb13715820767eae33a6daa0edf82f34 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:06:41 -0400
Subject: [powerpc] kexec: kexec_sequence() is in misc_64.S

Message-id: <1410545655-205645-173-git-send-email-dzickus@redhat.com>
Patchwork-id: 94181
O-Subject: [RHEL7 PATCH 172/626] powerpc/kexec: kexec_sequence() is in misc_64.S
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit 07fb41a7525539d7ad37c25f2a2689fd95a6ab68
Author: Geert Uytterhoeven <geert@linux-m68k.org>
Date:   Mon Oct 14 09:40:16 2013 +0200

    powerpc/kexec: kexec_sequence() is in misc_64.S

    Correct reference to the location of the kexec_sequence() assembly helper.
    There never was a kexec_stub.S in mainline.

    Signed-off-by: Geert Uytterhoeven <geert@linux-m68k.org>
    Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/kernel/machine_kexec_64.c b/arch/powerpc/kernel/machine_kexec_64.c
index 611acdf..be4e6d6 100644
--- a/arch/powerpc/kernel/machine_kexec_64.c
+++ b/arch/powerpc/kernel/machine_kexec_64.c
@@ -312,7 +312,7 @@ static union thread_union kexec_stack __init_task_data =
  */
 struct paca_struct kexec_paca;
 
-/* Our assembly helper, in kexec_stub.S */
+/* Our assembly helper, in misc_64.S */
 extern void kexec_sequence(void *newstack, unsigned long start,
       void *image, void *control,
       void (*clear_all)(void)) __noreturn;
-- 
1.7.1